tests/test.py

```diff
@@ -1,132 +0,0 @@
-import glob
```

Comment:
Created https://github.com/minishift/minishift-centos-iso/pull/163.

Reply:
I know... but had to move quickly

Reply:
merged... will rebase...